node/pkg/watchers/evm/connectors/logpoller.go

```diff
@@ -33,7 +33,7 @@ type LogPollConnector struct {
 func NewLogPollConnector(ctx context.Context, baseConnector Connector, client *ethClient.Client) (*LogPollConnector, error) {
 	connector := &LogPollConnector{Connector: baseConnector, client: client}
 	// The supervisor will keep the poller running
-	err := supervisor.Run(ctx, "logPoller", connector.run)
+	err := supervisor.Run(ctx, "logPoller", common.WrapWithScissors(connector.run, "logPoller"))
```

Comment:
don't we want the whole watcher to restart / the panic to propagate up?

Reply:
I was trying to not change existing behavior which currently just restarted the poller (assuming just a error was generated verses a panic).  I am open to letting it restart the whole watcher